README.md

```diff
@@ -34,7 +34,7 @@ yarn global add gluon-build
 
 ## Documentation
 
-Documentation is available on [Github pages](https://lepton-browser.github.io/build/) or in the docs folder of this repository.
+Documentation is available on [docs.gluon.dev](https://docs.gluon.dev) or in the docs branch of this repository.
```

Comment:
This is the correct link, but docs are still stored in the `docs/` folder of the main branch. The `docs` branch was a number of improvements I made to the docs that I still need to finalize before merging into main.

Thanks for your contribution though

Reply:
Ah, ok